### ISSUE TITLE
Caption background / Oat background fail contrast ratio

### ISSUE THREAD

**Issue by pjlee11** [bbc/simorgh#879]
**Describe the bug**
The `oat` background currently fails contrast ratio where there is a caption. The caption background as a foreground over the oat coloured body fails when looking at contrast ratio.

**Expected behaviour**
This should not fail colour contrast ratio

**Screenshots**
![pasted_image_at_2018-10-31__12_43_pm](https://user-images.githubusercontent.com/7791726/47789026-5679c600-dd0b-11e8-9c78-4ef3c24e855b.png)
![image 3](https://user-images.githubusercontent.com/7791726/47789027-57125c80-dd0b-11e8-8cc0-faed38a57a73.png)

**Additional context**
UX will need to supply a different colour that meets colour contrast ratio

- [x] Initially labelled with ["bug"](https://github.com/BBC-News/simorgh/labels/bug)

**Comment by bcmn** [comment]
@greenc05 clarified this elsewhere:

> the caption text is #404040 and the caption background is #D5D0CD which has a contrast ratio of 6.78:1, how does the page oat background colour affect this?
> If you are comparing the background of the caption with oat background that’s not content and doesn’t apply (edited)
> it’s just content that has to pass colour contrast ratios e.g. text, icons etc

Assuming UX are also happy with this, I think we can close the ticket.

**Comment by bcmn** [comment]
Just to confirm, I clarified this with UX, & they're on the same page as Catharine. As such, I'm closing this issue.
